MetaMask/metamask-extension
# Add transaction security check toggle

## Explanation
Add transaction security check toggle option.


## More Information
* Fixes #16257


## Screenshots/Screencaps



### Before

https://user-images.githubusercontent.com/92310504/197784279-2027edbf-08df-4fdd-b0a1-d32c5f04293a.mov




### After

https://user-images.githubusercontent.com/92310504/197784305-b4ed22ea-5ea0-4aab-924b-a204a6d509b6.mov




## Manual Testing Steps
Go to `Settings > Experimental` and check `Transaction security check` at the top.
